future_challenges_for_RSEng.tex

```diff
@@ -412,16 +412,17 @@ \subsection{Ethics and social considerations}
 
 \begin{story}{The greater world of Kim and Kay}{ethics}
 10 years have passed since Kim and Kay started on their journey of being RSEs. Back then,
-at the universities of Eden and Utopia, life was more orderly and harmonious. Kay is now working on a project that develops
-software for cheap water monitoring devices in the Global South. Kim is currently designing the software service stack
-at the university, and increasingly struggles to find compelling Open Source alternatives that enables them keep complete control of their data
-rather than relying on software and tools that are backed by large corporate entities. So far she still has the support of the university board, since it values
-strategic autonomy.
-Her son Kyle on the other hand started out with a degree in Physics and then took one of the newly created RSE Master's courses,
-but quickly decided that work in academia with its time limited contracts is not attractive.
-Governments around the world are struggling to reposition themselves in this emerging de-globalised world with its new blocks
-and therefore have de-prioritised publicly funded research. Kyle decided to work for a defence company since they offer
-more secure employment in this polarised world.
+at the universities of Eden and Utopia, life was more orderly and harmonious.
+Kim is currently designing the software service stack at the university.
+She increasingly struggles to find compelling Open Source alternatives that enables them to keep complete control of their data
+rather than relying on software and tools that are backed by large, foreign corporate entities.
+So far, she still has the support of the university board, since it values strategic autonomy.
+Kay used to work on a project that developed
+software for cheap water monitoring devices in the Global South, but now funding has dried up.
+Having gone through another fixed term contract he is now fed up
+with work in academia and decides that its time limited contracts are not attractive.
+Now Governments around the world are struggling to reposition themselves in this emerging de-globalised world with its new blocks and therefore have de-prioritised publicly funded research.
+Kay sees the writings on the wall and decides to work for a defence company since they offer more secure employment in this polarised world.
```

Comment:
Do we actually want to draw such a pessimistic picture of the future?
We could formulate it in a positive way:

> Kay originally worked for industrial projects, because of unreliable funding. But now, with longer-term positions at a central RSE department, and appreciation of sustainable software engineering as a vital component of research, Kay now works for a university, designing software for an affordable water monitoring system used in the global south.

I like the "works on something water-related, but funding has now dried up", but I would not form a pessimistic story around this.

Reply:
I agree, we might as well go for utopia, reality is grim enough without us adding to it. What about

> Kay had left academia to work in an industrial R&D department because of a more reliable employment. There he gained valuable experience working in an industrial setting. However, he was very happy when an opportunity arose to work in a central RSE department with an open-ended contract.  Apart from his main responsibility of developing and maintaining software for an affordable water monitoring system used in the global south, he now also teaches undergraduates digital literacy.